__tests__/app.test.js

```diff
@@ -74,3 +72,30 @@ describe("GET /api/reviews/:review_id", () => {
 			});
 	});
 });
+
+describe("GET /api/reviews", () => {
+	test("a reviews array of review objects including key  comment_count which is the total count of all the comments with this review_id. reviews should be sorted by date in descending order.there should not be a review_body property present on any of the review objects", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then((response) => {
+				response.body.reviews.forEach((review) => {
+					expect(typeof review.owner).toBe("string");
+					expect(typeof review.title).toBe("string");
+					expect(typeof review.category).toBe("string");
+					expect(typeof review.review_img_url).toBe("string");
+					expect(typeof review.created_at).toBe("string");
+					expect(typeof review.designer).toBe("string");
+					expect(typeof review.review_id).toBe("number");
+					expect(typeof review.votes).toBe("number");
+					expect(typeof review.comment_count).toBe("string");
+					expect(typeof review.review_body).toBe("undefined");
```

Comment:
we would never test for undefined. You could change this to expecting a property to not exist.